cli/flox-rust-sdk/src/models/floxmetav2/mod.rs

```diff
@@ -54,15 +59,23 @@ impl FloxmetaV2 {
         flox: &Flox,
         pointer: &ManagedPointer,
     ) -> Result<Self, FloxmetaV2Error> {
-        let host = flox.floxhub_host.as_str();
         let token = flox.floxhub_token.as_deref();
 
-        let git_options = floxmeta_git_options(&flox.floxhub_host, token);
+        let mut floxhub = Floxhub::new(pointer.floxhub_url.to_owned());
+        if let Some(git_url_override) = &pointer.floxhub_git_url_override {
+            floxhub.set_git_url_override(git_url_override.clone());
+        }
+
+        let git_url = floxhub
+            .git_url()
```

Comment:
question nonblocking: every time you call this you map_err, is there a reason you don't do that inside `git_url()`? Because it doesn't live in the floxmetav2 module?

Reply:
changing the hostname of the base url is a fallible operation (if the base url was an IP address or a file (i.e. no host).
It's probably better to return a `FloxhubError` instead of the raw `Url::ParseError` but it has to be handled anyway `Floxhub::git_url` is called